friends/src/Components/FriendForm.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+class FriendForm extends React.Component {
+  state = {
+    friend: {
+      name: '',
+      age: '',
+      email: '',
+    }
+  };
+
+  changeHandler = ev => {
+    ev.persist();
+    let value = ev.target.value;
+
```

Comment:
Since age is number in our data, and you are storing it as a string, I would convert it to a number and then store it for data.